README.md

```diff
@@ -136,17 +136,18 @@ Interoperate multiple service meshes with service mesh adapters provision, confi
 
 <p style="clear:both;">
 <a href="https://raw.githubusercontent.com/layer5io/meshery/master/docs/assets/img/readme/meshery_lifecycle_management.png"><img alt="Layer5 Service Mesh Configuration Management" src="docs/assets/img/readme/meshery_lifecycle_management.png"  style="margin-right:10px;margin-bottom:10px;" width="45%" align="left"/></a>
+  <br /><br /><br /><br />
 <h3>Service Mesh Configuration Management</h3>
 
 Assess your service mesh configuration against deployment and operational best practices with Meshery's configuration validator.
-
 Onboard your workload onto the service mesh with confidence. Check your service mesh configuration for anti-patterns and avoid common pitfalls.
 <br /><br /><br /><br />
 
 </p>
 
-<h3>Adhering to Service Mesh Standards</h3>
 
+<h3>Adhering to Service Mesh Standards</h3>
+<img alt="Meshery" src="https://i.ibb.co/LzXS7WV/smp-dark.png" style="margin-left:10px;" width="18%" align="left" />
```

Comment:
Thanks for this PR, @Aarush-Goyal. 

It would be great to have the logo with "SMP" in it, if the images fits into the section.

Will you store a copy of the logo in this directory - `https://github.com/layer5io/meshery/tree/master/docs/assets/img/readme`?

Reply:
The alt-text should also be changed to "SMP" @Aarush-Goyal